BUILDING.txt

```diff
@@ -29,7 +29,11 @@ This requires a recent version of docker (1.4.1 and higher are known to work).
 On Linux / Mac:
     Install Docker and run this command:
 
-    $ ./start-build-env.sh
+    $ ./start-build-env.sh [OS platform]
+
+   - [OS Platform] One of [centos_7, centos_8, debian_10, ubuntu_20, ubuntu_24, windows_10].
+        Default is 'ubuntu_20'.
+        Note: Currently only  default ('ubuntu_20') is supported on arm machine
```

Comment:
nit: remove an extra space before "default"

Just out of curiosity, have you encountered any issues with supporting ubuntu_24 on ARM, the support might be worth doing, given that a lot of developers use Apple Silicon

Reply:
Supporting ARM in ubuntu-22+ needs some amount of effort. Ex: `phantomjs` library is not available for arm in recent OS distributions.

I will try to address that in a follow up PR.

Thanks for checking.

Reply:
SGTM, thank you for sharing that.